--- a/keylime-agent/Cargo.toml	2024-01-31 10:25:42.291841679 +0100
+++ b/keylime-agent/Cargo.toml	2024-01-31 10:28:02.795282892 +0100
@@ -48,18 +48,6 @@
 default = []
 # this should change to dev-dependencies when we have integration testing
 testing = ["wiremock"]
-# Whether the agent should be compiled with support to listen for notification
-# messages on ZeroMQ
-#
-# This feature is deprecated and will be removed on next major release
-with-zmq = ["zmq"]
-# Whether the agent should be compiled with support for python revocation
-# actions loaded as modules, which is the only kind supported by the python
-# agent (unless the enhancement-55 is implemented). See:
-# https://github.com/keylime/enhancements/blob/master/55_revocation_actions_without_python.md
-#
-# This feature is deprecated and will be removed on next major release
-legacy-python-actions = []
 
 [package.metadata.deb]
 section = "net"
